packages/lib/src/permissions.ts

```diff
@@ -484,6 +484,22 @@ definePermission({
 	title: "Delete user",
 	description: "Delete users and all data stored for them.",
 });
+definePermission({
```

Comment:
All of these effectively give cluster admin rights, right?

Reply:
These permissions allow for the "bulk" request. To actully set/create users you still require the other permissions. The UI is setup to refect this. I will add this to the description of the permissions.